src/Error/ExceptionTrap.php

```diff
@@ -104,14 +110,25 @@ public function renderer(Throwable $exception, $request = null)
             }
 
             /** @var \Cake\Error\ExceptionRendererInterface $instance */
-            $instance = new $class($exception, $request);
+            $instance = new $class($exception, $request, $this->_config);
```

Comment:
Do we want to pass the entire config here or just renderer options?

Reply:
The logger also gets all the options, so I thought passing it all would be more consistent.

Reply:
Just checking. In some other places, we've had dense code pass option arrays around instead of the 1 value it needed.